index.html

```diff
@@ -1,116 +1,129 @@
+              <i class="fab fa-linkedin-in fa-lg"></i>
+            </a>
+          </li>
+          <li class="nav_item">
+            <a href="https://twitter.com/imsaadahmad" target="_blank">
+              <i class="fab fa-twitter fa-lg"></i>
+            </a>
+          </li>
+        </ul>
+      </div>
+
+      <div class="burger">
+        <i class="fas fa-bars fa-lg"></i>
+      </div>
     </nav>
 
     <main class="container_main">
-        <div class="main_top">
-            <div class="main_top__left">
-                <h1>Build your site at lightening fast speed with <span class='library_name'>Chroma</span></h1>
-                <p>Don't let design take your mental space. Use Chroma to design wesites on the go.</p>
-                <div id="cta_buttons">
-                    <a class="btn btn_lg btn_outline" href="#installation">Installation</a>
-                    <a class="btn btn_lg btn_primary" href="./documentation.html">Documentation</a>
-                </div>
-            </div>
-            <div class="main_top__right">
-                <img src="./assets/chroma landing.svg" alt="image" class="image_main">
-            </div>
+      <div class="main_top flex_align__center">
+        <div class="main_top__left">
+          <h1>
+            Build your site at lightening fast speed with
+            <span class="library_name">Chroma</span>
+          </h1>
+          <p>
+            Don't let design take your mental space. Use Chroma to design
+            wesites on the go.
+          </p>
+          <div id="cta_buttons">
+            <a class="btn btn_lg btn_outline" href="#installation">
+              Installation
+            </a>
+            <a class="btn btn_lg btn_primary" href="./documentation.html">
+              Documentation
+            </a>
+          </div>
         </div>
+        <div class="main_top__right">
+          <img
+            src="./assets/chroma landing.svg"
+            alt="image"
+            class="image_main" />
+        </div>
+      </div>
 
-        <div class="main_bottom" id="installation">
-            <div class="main_bottom__left">
-                <h1>Installation</h1>
-                <p>Lorem ipsum dolor sit amet consectetur, adipisicing elit. Ratione iure nobis, magnam consequatur ipsum quibusdam reiciendis nemo inventore nisi </p>
-            </div>
-
-            <div class="main_bottom__right">
-
-            </div>
+      <div class="main_bottom" id="installation">
+        <div class="main_bottom__up flex_align__center flex_col">
+          <h1>Installation</h1>
+          <p>
+            Paste the stylesheet link in your header tag before any stylesheet
+            to use the styles provided by of chroma.
+          </p>
         </div>
+        <div class="main_bottom__down flex_justify__center">
+          <div class="code_snippet">
+            <script src="https://gist.github.com/saadahmadhere/67d1ff97df32f36a25de16262d5bad26.js"></script>
```

Comment:
Check Indentation

Reply:
I have set the default indentation to 2 spaces. Everything is fine according to me.